iceoryx_hoofs/test/moduletests/test_unix_domain_sockets.cpp

```diff
@@ -77,14 +77,15 @@ class UnixDomainSocket_test : public Test
 
     bool createTestSocket(const UnixDomainSocket::UdsName_t& name)
     {
+        bool socketCreationSuccess = true;
+#if !defined(_WIN32)
```

Comment:
I assumed that unix domain sockets are not used for Windows. Can't we just disable all the tests for Windows?